regression/cbmc/posix_memalign/main.c

```diff
@@ -0,0 +1,17 @@
+#include <malloc.h>
```

Comment:
This header isn't portable (as CI reminded us), but also not necessary. Please just remove that line.

Reply:
I addressed the requested changes in 3a11d7d